src/app/genes/genes-list/genes-list.component.html

```diff
@@ -116,7 +116,7 @@ <h2 class="vmi-wrapper full-width no-margin-top small-margin-left">
             </tr>
         </ng-template>
         <ng-template pTemplate="body" let-rowData let-columns="columns">
-            <tr [pSelectableRow]="rowData" tabindex="0">
+            <tr [pSelectableRow]="rowData">
```

Comment:
Tab sort didn't seem to be working here, and wasn't in similar-genes, so I just removed it.

Reply:
If it doesn't impact usability for non-mouse users then I don't have a problem with removing it.